KsApi/PKCETest.swift

```diff
@@ -0,0 +1,20 @@
+@testable import KsApi
+import XCTest
+
+final class PKCETest: XCTestCase {
```

Comment:
Some rough tests - we'll coordinate more examples later, to make sure iOS + Android have the same behavior.